pkg/apis/v1alpha1/awsnodetemplate.go

```diff
@@ -34,6 +34,9 @@ type SecurityGroup struct {
 	// ID of the security group
 	// +required
 	ID string `json:"id"`
+	// Name of the security group
+	// +optional
+	Name string `json:"name"`
```

Comment:
```suggestion
	Name string `json:"name,omitempty"`
```

Should this be `omitempty` if the value is `""`?